src/modules/appointments/appointments.service.ts

```diff
@@ -76,4 +78,86 @@ export class AppointmentsService {
 
     return appointments;
   }
+
+  async getFutureAppointmentsByDoctorId(
+    id: number,
+    pagination: PaginationOptionsDto,
+  ): Promise<Appointment[]> {
+    const now = moment().toDate();
+
+    const futureAppointments = await this.appointmentRepository
```

Comment:
are you count user timezone?

Reply:
yes, i'm using create in utc and getting timezone from browser